### PR TITLE
Assignments/array hofs

### PR DESCRIPTION
**Pull request by Eliass-m**

### CODE REVIEW

File: assignments/array-hofs/code.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const evens = (array) => {
+  return array.filter((n) => n % 2 === 0)
+}
+
+const odds = (array) => {
+  return array.filter((n) => n % 2 !== 0)
+}
+
+const big = (array) => {
+  return array.filter((n) => n / 100 > 1)
```
Comment (gigamonkey) [inline assignments/array-hofs/code.js:10]: dunno which of the three of you came up with this. it works but it's a funny way to write `n > 100`

**Review by gigamonkey — CHANGES_REQUESTED** [review]
Good start. Keep going.

File: assignments/array-hofs/code.js (hunk 1 above)
Comment (Eliass-m) [reply assignments/array-hofs/code.js:10]: theo

**Review by gigamonkey — CHANGES_REQUESTED** [review]
Almost there! Just one to go. Everything else looks fine.

**Review by gigamonkey — APPROVED** [review]
Looks good.

**Review by gigamonkey — APPROVED** [review]
The ones you've done look good.

**Pull request merged**